test/unit/lib/redcap_test.rb

```diff
@@ -17,15 +17,15 @@ class Redcap < ActiveSupport::TestCase
     assert_equal 5, forms.values.length
 
     rscount = ResponseSet.count
-    qcount = Question.count
+    qcount  = Question.count
     formcount = Form.count
 
     forms.each_pair do |_k, v|
       v.save
     end
 
-    assert_equal rscount + 67, ResponseSet.count
-    assert_equal qcount + 198, Question.count
+    assert_equal rscount + 67,  ResponseSet.count
```

Comment:
Maybe move these hard coded numbers (67, 198, 5) into variables?

If we change our redcap import data later it'd be nice to just go to the top of the relevant file to change things.